packages/eslint-plugin-pf-codemods/test/rules/v5/applicationLauncher-updated-params.js

```diff
@@ -0,0 +1,90 @@
+const ruleTester = require('../../ruletester');
+const rule = require('../../../lib/rules/v5/applicationLauncher-updated-params');
+const propNames = ['onFavorite', 'onSearch'];
+
+const ruleRunner = (prop) => {
```

Comment:
could we just do `propNames.forEach(  (prop) => {` directly? so I don't have to wonder what's going on until i scroll to the bottom 😆

Reply:
What's wrong with having some adventure every and then? 😂

Reply:
That being said, given the temporary nature of codemods I wonder a bit about extracting these event param tests into a reusable helper function.

I'd typically be against that sort of thing since it decreases test readability, but that tradeoff would probably be worth doing for codemods since we won't be maintaining them long term and it would speed up development.

WDYT?

Reply:
if it's set up right and we trust the helpers, it shouldn't reduce readability.. might make sense to have another helper in the tests dir, up to you if you wanna take a shot at that